provider/testdata/sidecar_testdata.toml

```diff
@@ -1,7 +1,13 @@
 [frontends]
     [frontends.web]
-    backend = "web"
-    passHostHeader = true
-    entrypoints = ["http","https"]
+        backend = "web"
+        passHostHeader = true
+        entrypoints = ["http","https"]
         [frontends.web.routes.test_1]
-        rule = "Host: some-aws-host"
+            rule = "Host: some-aws-host"
+        [frontends.web.maxconn]
```

Comment:
Should there be another test without the frontends.web.maxconn to verify the default Maxconn setting?